TransferMethod/Sources/SelectTransferMethodTypePresenter.swift

```diff
@@ -310,14 +310,33 @@ final class SelectTransferMethodTypePresenter {
     }
 
     private func loadCurrency(_ keys: HyperwalletTransferMethodConfigurationKey?) {
-        guard let firstCurrency = keys?.currencies(from: selectedCountry)?.first,
-            let currencyCode = firstCurrency.code else {
-            view?.showAlert(message: String(format: "no_currency_available_error_message".localized(), selectedCountry))
-            return
+        guard let countries = keys?.countries(),
+              let country = countries.first(where: { $0.code == selectedCountry }) else {
+               // Handle the case when country is not found
+               let errorMessage = String(format: "no_country_available_error_message".localized(), selectedCountry ?? "Unknown Country")
```

Comment:
Why do we need `"Unknown Country"`?   
Can we just use 
`view?.showAlert(message: String(format: "no_currency_available_error_message".localized(), selectedCountry))`